internal/service/fw/model_atcfw_internal_domains.go

```diff
@@ -0,0 +1,144 @@
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/infobloxopen/bloxone-go-client/fw"
+
+	"github.com/infobloxopen/terraform-provider-bloxone/internal/flex"
+)
+
+type AtcfwInternalDomainsModel struct {
+	CreatedTime     timetypes.RFC3339 `tfsdk:"created_time"`
+	Description     types.String      `tfsdk:"description"`
+	Id              types.Int64       `tfsdk:"id"`
+	InternalDomains types.List        `tfsdk:"internal_domains"`
+	IsDefault       types.Bool        `tfsdk:"is_default"`
+	Name            types.String      `tfsdk:"name"`
+	Tags            types.Map         `tfsdk:"tags"`
+	UpdatedTime     timetypes.RFC3339 `tfsdk:"updated_time"`
+}
+
+var AtcfwInternalDomainsAttrTypes = map[string]attr.Type{
+	"created_time":     timetypes.RFC3339Type{},
+	"description":      types.StringType,
+	"id":               types.Int64Type,
+	"internal_domains": types.ListType{ElemType: types.StringType},
+	"is_default":       types.BoolType,
+	"name":             types.StringType,
+	"tags":             types.MapType{ElemType: types.StringType},
+	"updated_time":     timetypes.RFC3339Type{},
+}
+
+var AtcfwInternalDomainsResourceSchemaAttributes = map[string]schema.Attribute{
+	"created_time": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "The time when this Internal Domain lists object was created.",
+	},
+	"description": schema.StringAttribute{
+		Optional:            true,
+		Default:             stringdefault.StaticString(""),
+		Computed:            true,
+		MarkdownDescription: "The brief description for the internal domain lists .",
+	},
+	"id": schema.Int64Attribute{
+		Computed: true,
+		PlanModifiers: []planmodifier.Int64{
+			int64planmodifier.UseStateForUnknown(),
+		},
+		MarkdownDescription: "The Internal Domain object identifier.",
+	},
+	"internal_domains": schema.ListAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "The list of internal domains, should be unique to each other and has to be read-only from the API level.",
+	},
+	"is_default": schema.BoolAttribute{
+		Computed:            true,
+		MarkdownDescription: "True if name is 'Default Bypass Domains/CIDRs' otherwise false.",
+	},
+	"name": schema.StringAttribute{
+		Optional: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
+		MarkdownDescription: "The name of the internal domain lists.",
+	},
+	"tags": schema.MapAttribute{
+		ElementType:         types.StringType,
+		Optional:            true,
+		MarkdownDescription: "Enables tag support for resource where tags attribute contains user-defined key value pairs",
```

Comment:
It might be good to use the same description we use for tags everywhere else.

Reply:
fixed